bot.go

```diff
@@ -105,6 +109,19 @@ func newMessage(s *discordgo.Session, m *discordgo.MessageCreate) {
 		sendMessage(s, m.ChannelID, textapis.CatFact())
 	case "!swanson":
 		sendMessage(s, m.ChannelID, textapis.Swanson())
+	case "!roll":
+		elems := strings.Split(args[1], "d")
+		reps, err := strconv.Atoi(elems[0])
+		if err != nil {
+			fmt.Println("Error parsing dice count: ", err)
+			return
```

Comment:
Might want to respond in chat with error message in these cases. Otherwise this PR LGTM.